.github/workflows/release.yml

```diff
@@ -41,22 +52,6 @@ jobs:
           echo Logging into repository $CODE_ARTIFACT_REPO
           aws codeartifact login --tool npm --repository $CODE_ARTIFACT_REPO --domain awsui --domain-owner ${{ secrets.AWS_ACCOUNT_ID }} --region us-west-2 --namespace @cloudscape-design
```

Comment:
I moved code artifact login to the last step in the build, because all steps above do not need these credentials anymore. So, let's reduce their scope

Git could not detect the change properly and shows that I moved other steps instead